draft-irtf-cfrg-opaque.md

```diff
@@ -1263,42 +1273,34 @@ such that the following conditions are met:
   OPRF(P-256, SHA-256).
 - The KDF, MAC, and Hash functions implement the interfaces in {{dependencies}}.
   Examples include HKDF {{RFC5869}} for the KDF, HMAC {{!RFC2104}} for the MAC,
-  and SHA-256 and SHA-512 for the Hash functions.
+  and SHA-256 and SHA-512 for the Hash functions. If an extensible output function
+  such as SHAKE128 {{FIPS202}} is used then the output length `Nh` MUST be chosen
+  to align with the target security level of the OPAQUE configuration. For example,
+  if the security parameter is 128-bits, then `Nh` SHOULD be at least 32 bytes.
```

Comment:
This is the second part of this change.

Reply:
What is `security parameter` referring to?

Reply:
Clarified that it's the target security parameter for the OPAQUE configuration. We're not (yet) crisp about that, but we can always expand on that later.